Lib/zipfile.py

```diff
@@ -8,6 +8,7 @@
 import importlib.util
 import io
 import os
+import pathlib
```

Comment:
I think including pathlib might be problemmatic, mainly because of the added dependency on the backport. I encountered that [same concern here](https://github.com/jaraco/zipp/pull/10#issuecomment-511134529) and [here](https://github.com/jaraco/zipp/pull/11).

Reply:
I am working on putting together the 3 options on solving this as suggested by @ericvsmith below.

Reply:
I have removed pathlib in my latest update.